src/cmdstan-guide/optimize_config.qmd

```diff
@@ -33,21 +104,69 @@ of default configuration options:
 #     jacobian = 0 (Default)
 #     iter = 2000 (Default)
 #     save_iterations = 0 (Default)
+...
+```
+
+Following the config information, are two lines of output:
```

Comment:
This is a non-Oxford comma situation, so it should be "two lines of output, X and Y".

Reply:
I promise I was just copying these from the existing files, I am not _this_ bad with commas.

Regardless, fixed.